Libs/Widgets/ctkConsole.h

```diff
@@ -175,6 +176,20 @@ class CTK_WIDGETS_EXPORT ctkConsole : public QWidget
 
   static QString stdInRedirectCallBack(void * callData);
 
+  /// Get the list of shortcuts that trigger the completion popup
+  /// \sa setCompleterShortcuts(), addCompleterShortcut()
+  QList<QKeySequence> completerShortcuts()const;
+
+  /// Set the list of shortcuts for the completion to popup
+  /// Default is simply Qt::Key_Tab.
+  /// \sa completerShortcuts(), addCompleterShortcut()
+  void setCompleterShortcuts(const QList<QKeySequence>& keys);
+
+  /// Convinience method to a key sequence to the list of shortcuts that
```

Comment:
`Convinience method to a key sequence` -> `Convenience method to add a key sequence`